usl_pipeline/study_area_uploader/study_area_uploader/main.py

```diff
@@ -153,6 +165,14 @@ def parse_args() -> argparse.Namespace:
         + " to CityCat storage bucket.",
         action=argparse.BooleanOptionalAction,
     )
+    parser.add_argument(
+        "--upload-chunks",
```

Comment:
What's a case where the user wouldn't want to upload chunks? I think we always want to upload them.

Reply:
Agreed. Removed it. And did the same for `build_chunks` one.